### PR TITLE
Carthage update from master

### PR DESCRIPTION
**Pull request by mnaamani**
Merging master branch into carthage, 
dropping commented out code related to building for raspberry-pi armv7 (32-bit)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1203175606390796) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by Lezek123 — COMMENTED** [review]
Looks good, however I noticed that we're still using outdated version for some GH actions: 
image

Is there a specific reason for this, or were they just overlooked?

**Comment by mnaamani** [comment]
> Is there a specific reason for this, or were they just overlooked?

Maybe overlooked, I'll double check.

File: .github/workflows/run-network-tests.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```
Comment (Lezek123) [inline .github/workflows/run-network-tests.yml]: I think this should be `v3` to match the other updates:
```suggestion
    - uses: actions/checkout@v3
```

File: .github/workflows/run-network-tests.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
...
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
```
Comment (Lezek123) [inline .github/workflows/run-network-tests.yml]: This one was also updated to `v3` in workflows like [.github/workflows/joystream-cli.yml](https://github.com/Joystream/joystream/pull/4379/files#diff-81c9870c963d108d2e27f1ab912ca133e2b62feca653ff14f0de1266f098c5d4):
```suggestion
      uses: actions/setup-node@v3
```

File: .github/workflows/run-network-tests.yml [hunk 3]
```diff
@@ -34,7 +34,7 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```
Comment (Lezek123) [inline .github/workflows/run-network-tests.yml]:
```suggestion
    - uses: actions/checkout@v3
```

File: .github/workflows/run-network-tests.yml [hunk 4]
```diff
@@ -34,7 +34,7 @@ jobs:
...
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
```
Comment (Lezek123) [inline .github/workflows/run-network-tests.yml]:
```suggestion
      uses: actions/setup-node@v3
```

**Review by Lezek123 — CHANGES_REQUESTED** [review]

**Review by Lezek123 — APPROVED** [review]
LGTM if the CI checks pass

**Pull request merged**